### DISCUSSION TITLE
Timestamp missing in Json report

### DISCUSSION THREAD

**Discussion by priya-dsp** [artilleryio/artillery#2741] [Artillery]
Hi, 
When we are generating Json report for Artillery yaml scripts, we don't find timestamp included in that JSOn report. Could you please guide how to configure Json report to get timestamp added? Below is sample JSon how we need it. 
![image](https://github.com/artilleryio/artillery/assets/159108432/9f0c5fc2-9cd3-476b-a8db-d609fa86c38a)

**Comment by hassy** [comment]
hi @priya-dsp 👋 the screenshot looks like it's a report from Artillery v1. In Artillery v2 the timestamp is available under the `period` field, and is set to the number of milliseconds elapsed since the epoch (same as `Date.now()` in JS). It can be converted to a readable timestamp with `new Date(value)`.

**Reply by priya-dsp** [reply]
Is period mentioned in Json report for total test duration or do we get for every seconds?  We are looking for continuous "response time over time" graph for which we need period.
